### PR TITLE
Various Color Fixes

### LINKED ISSUE #2305: Inconsistent Coloring in Involvements/Faculty Pages
There are some messy color choices that lead to some insufficient color contrasts on multiple pages.
- Involvements Club Page

Image

Image

- Faculty/Staff Profiles

Image

### PR DESCRIPTION
**Pull request by Luke-W-Hart**
Fixes #2305 
How it looks now
Screenshot from 2024-06-11 16-32-57
Screenshot from 2024-06-11 16-33-11
Screenshot from 2024-06-11 16-45-06

### CODE REVIEW

**Review by EjPlatzer — CHANGES_REQUESTED** [review]
Looks good! Just one question.

File: src/views/InvolvementProfile/components/Membership/components/MemberList/index.jsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ const MemberList = ({
           </Grid>
         </Grid>
       }
-      className={styles.headerStyle}
+      className="gc360_header"
```
[the remark this answers is not in the document]
Comment (Luke-W-Hart) [reply src/views/InvolvementProfile/components/Membership/components/MemberList/index.jsx:89]: It's not anymore after my last commit which fixes the color for mobile and admin views. It's the only thing left in the MemberList.module.scss file, should I delete it?

[the remark this answers is not in the document]
Comment (EjPlatzer) [reply src/views/InvolvementProfile/components/Membership/components/MemberList/index.jsx:89]: Yeah, if the scss file isn't being used feel free to delete it. Make sure to remove the import from the .jsx file also.

[the remark this answers is not in the document]
Comment (Luke-W-Hart) [reply src/views/InvolvementProfile/components/Membership/components/MemberList/index.jsx:89]: Awesome, thanks Evan!

**Review by EjPlatzer — APPROVED** [review]
LGTM!

**Pull request merged**